packages/ses/package.json

```diff
@@ -36,11 +36,13 @@
   "exports": {
     ".": {
       "import": "./index.js",
-      "require": "./dist/ses.cjs"
+      "require": "./dist/ses.cjs",
+      "types": "./types.d.ts"
```

Comment:
Can you add a line to `package.json.md` explaining this?

Reply:
@kriskowal Done.  Also fixed another path in there that was wrong.